src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/VilkårTabellRadEndre.tsx

```diff
@@ -27,7 +27,7 @@ import {
 } from '../../../../context/Vilkårsvurdering/VilkårsvurderingContext';
 import type { IBehandling } from '../../../../typer/behandling';
 import { BehandlingÅrsak } from '../../../../typer/behandling';
-import type { IGrunnlagPerson } from '../../../../typer/person';
+import { type IGrunnlagPerson, PersonType } from '../../../../typer/person';
```

Comment:
Tror ikke dette er riktig måte å skrive det på 🤔  Ganske sikker på at "type" skal være utenfor krøllparantesen. Burde derfor importere PersonType i egen import uten "type" foran

Reply:
Åh, var sånn eslinten min ble fornøyd 😓  haha

Reply:
Hmmm merkelig, for jeg får en error i intellij hvis det er skrevet på den måten 🤔